src/utils/validation.js

```diff
@@ -98,7 +98,12 @@ const editCourseValidate = (values, props) => {
     const { key: targetKey } = targetRun;
     const isSubmittingRun = run.key === targetKey;
     if (isSubmittingRun) {
-      const runRequiredFields = ['transcript_languages', 'staff'];
+      // naive check to deteremine if track is executive education or not
+      const isExecutiveEducation = registeredFields
+        && registeredFields['prices.executive-education']
+        && registeredFields['prices.executive-education'].count;
```

Comment:
I guess the count condition isn’t necessary, it cannot be 0 if the `prices.executive-education` is set.

Reply:
While testing, I have observed that in `Edit Course Page` while we are still editing the form and change course enrollment track from audit to exec ed to audit again, we still see price for exec ed but with 0 count. Not having a count check will fail validation for audit only course as well. 

![image](https://user-images.githubusercontent.com/47540683/155131940-1dafe6bb-8272-4595-a777-a25e980eab60.png)